components/Tabs/src/MDCTabBar.m

```diff
@@ -264,6 +275,28 @@ + (CGFloat)defaultHeightForItemAppearance:(MDCTabBarItemAppearance)appearance {
   return [self defaultHeightForBarPosition:UIBarPositionAny itemAppearance:appearance];
 }
 
+- (void)setTitleColor:(nullable UIColor *)color forState:(UIControlState)state {
+  _titleColorsForState[@(state)] = color;
+  if (self.statefulAPIEnabled) {
```

Comment:
Should we also check that `state` == `currentState`?

Reply:
No, the underlying MDCItemBar handles that, we just tell it a selected and unselected state.